theano/gpuarray/dnn.py

```diff
@@ -234,6 +234,9 @@ def get_params(self, node):
             ptr = ctx.cudnn_handle.value
             res = handle_type.make_value(ptr)
             ctx.cudnn_handle_param = res
+        if isinstance(self.params_type, Wrapper) and self.params_type.has_type(handle_type):
```

Comment:
If it is a Wrapper and it don't have handle_type, you should raise an error. Currently params_type will just be ignored.